swagger.json

```diff
@@ -5850,6 +5850,72 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "User": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "The id of the User record.",
+                    "example": 123
+                },
+                "name" : {
+                    "type": "string",
+                    "description": "The first and last name of the user.",
+                    "example": "Jane Doe"
+                },
+                "email": {
+                    "type": "string",
+                    "description": "The email address of this user.",
+                    "example": "user@company.com"
+                },
+                "organizationRole" : {
+                    "type": "string",
+                    "description": "The name of the role the user is assigned to for the organization. This will be blank for users that only have access to specific tags.",
+                    "example": "Read-Only Admin"
+                },
+                "tagRoles" : {
+                    "type": "array",
+                    "description": "The specific tags this user has access to. This will be blank for users that have full access to the organization.",
+                    "items": {
+                        "$ref": "#/definitions/UserTagRole"
+                    }
+                },
+                "authType": {
+                    "type": "string",
+                    "enum": [
+                        "basic",
+                        "saml"
+                    ],
+                    "description": "The authentication type the user uses to authenticate. To use SAML your organization must have a configured SAML integration."
```

Comment:
Should we change this to `To use SAML this organization must have a configured SAML integration` since we don't use second person anywhere in the doc except in the intro line?

Reply:
I like this more